MAUI/Scheduler/day-week-views.md

```diff
@@ -41,6 +41,8 @@ this.Scheduler.DaysView.NumberOfVisibleDays = 3;
 {% endhighlight %}
 {% endtabs %}
 
+![visible-days-count-in-maui-scheduler](images/day-week-views/number-of-visibledays-in-maui-scheduler.png)
```

Comment:
visible-days-count-in-maui-scheduler-days-view

Reply:
Alt text updated.